### PR TITLE
Movie show page with reviews

### PR DESCRIPTION
**Pull request by zettlingzettel**
- added reviews to the show page
- created reviews table
- created associations
- updated serializer
- updated seeds

### CODE REVIEW

File: app/javascript/react/components/ReviewTile.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const ReviewTile = (props) => {
+  return (
+    <div>
+        <li>Rating: {props.rating}</li>
+        <p>Review: {props.body}</p>
+      </div>
+  )
+}
```
Comment (nwalberts) [inline app/javascript/react/components/ReviewTile.js:10]: div indentation is slightly off

File: app/serializers/review_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ReviewSerializer < ActiveModel::Serializer
+  # include FastJsonapi::ObjectSerializer
```
Comment (nwalberts) [inline app/serializers/review_serializer.rb:2]: no code comments!

File: config/routes.rb [hunk 3]
```diff
@@ -8,7 +8,8 @@
 
   namespace :api do
     namespace :v1 do
-      resources :movies, only: [:index, :show]
+      resources :movies, only: [:index, :show] 
+      # resources :reviews, only: [:show]
```
Comment (nwalberts) [inline config/routes.rb:12]: no code comments

File: db/migrate/20221103141808_create_reviews.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
+
```
Comment (nwalberts) [inline db/migrate/20221103141808_create_reviews.rb:2]: remove extra lines

**Comment by zettlingzettel** [comment]
Added tests

File: db/seeds.rb [hunk 5]
```diff
@@ -2115,4 +2115,10 @@
...
 The situation becomes complicated when a gang members falls in love with a rival's sister."
-)
\ No newline at end of file
+)
+test_user = User.create(email:'testpass@example.com', password: 'valid_password', password_confirmation: 'valid_password')
+
+review1 = Review.create(rating: "5", body: "Review1", movie_id: movie1.id, user_id: test_user.id)
+review2 = Review.create(rating: "4", body: "Review2", movie_id: movie1.id, user_id: test_user.id)
+review3 = Review.create(rating: "5", body: "Review3", movie_id: movie2.id, user_id: test_user.id)
+review4 = Review.create(rating: "2", body: "Review4", movie_id: movie2.id, user_id: test_user.id)
```
Comment (nwalberts) [inline db/seeds.rb:2124]: lets use some slightly more real data here. Not needed for the user, but lets do it for our reviews and movies

**Review by nwalberts — CHANGES_REQUESTED** [review]

**Pull request merged**